dts/bindings/pwm/ti,mspm0-pwm.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2024 Bang & Olufsen A/S, Denmark
+#
+# SPDX-License-Identifier: LicenseRef-BnOProprietary
+
+description: |
+  Driver to enable MSPM0 PWM hardware for TIMx timers (TIMA/TIMG).
+  This driver is implementing the existing Zephyr PWM API (pwm_driver_api) and
+  supports the set_cycles and get_cycles_per_sec functions.
+
+  For the set_cycles function, passing flags = 1 as the last argument will stop the running counter.
+  By default, making any call with flags = 0, will restart the counter
+
+compatible: "ti,mspm0-pwm"
+
+include: [base.yaml,pinctrl-device.yaml]
+
+properties:
+  ti,prescaler:
+    type: int
+    default: 0
+    description: Specifies the value used to divide the input clock frequency, allowing adjustment of the effective clock rate used by the timer.
+
+  ti,mode:
+    type: int
```

Comment:
You could use an enum instead, then you don't need the defines and you put a clear definition of the valid values. Can you handle if I put `7` here as an example?